packages/playground/remote/src/lib/boot-playground-remote.ts

```diff
@@ -232,6 +239,25 @@ export async function bootPlaygroundRemote() {
 		throw e;
 	}
 
+	/**
+	 * If the browser is online we can download WordPress assets asynchronously to speed up the boot process.
```

Comment:
```suggestion
	 * If the browser is online we download WordPress assets asynchronously to speed up the boot process.
```

Reply:
Done [2ef0bcd](https://github.com/WordPress/wordpress-playground/pull/1604/commits/2ef0bcdf0ac36f63927b18025ef96239b12509ce). I kept that part short because it's described in detail in the function docblock.